keyboards/santy65/info.json

```diff
@@ -0,0 +1,134 @@
+{
+    "manufacturer": "Santy",
+    "keyboard_name": "santy65",
+    "maintainer": "qmk",
+    "bootloader": "qmk-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [{ "pin_a": "B6", "pin_b": "C7" }]
+    },
+    "features": {
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
+        "encoder_map": true,
```

Comment:
This cannot be enabled at the keyboard level, as it depends on keymap specific configuration. So it should enabled only at the keymap level `rules.mk` file(s).
```suggestion
```